input/pagecontent/terminology.md

```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
+|[Immunisation Anatomical Site](https://healthterminologies.gov.au/fhir/ValueSet/immunisation-anatomical-site-1)|[AU Base Immunisation](StructureDefinition-au-immunization.html)|NCTS|
+|[Immunisation Route of Administration](https://healthterminologies.gov.au/fhir/ValueSet/immunisation-route-of-administration-1)|[AU Base Immunisation](StructureDefinition-au-immunization.html)|NCTS|
+|[Indicator of Hypersensitivity or Intolerance to Substance](https://healthterminologies.gov.au/fhir/ValueSet/indicator-hypersensitivity-intolerance-to-substance-2)|[AU Base Allergy Intolerance](StructureDefinition-au-allergyintolerance.html)|NCTS|
+|[Individual Healthcare Identifier Record Status](https://healthterminologies.gov.au/fhir/ValueSet/ihi-record-status-1)|[IHI Record Status](StructureDefinition-ihi-record-status.html)|NCTS|
+|[Individual Healthcare Identifier Status](https://healthterminologies.gov.au/fhir/ValueSet/ihi-status-1)|[IHI Record Status](StructureDefinition-ihi-record-status.html)|NCTS|
+|[Location Type (Physical) - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-location-physical-type-extended.html)|[AU Base Location](StructureDefinition-au-location.html)|AU Base|
+|[Medication Form](https://healthterminologies.gov.au/fhir/ValueSet/medication-form-1)|[AU Base Medication](StructureDefinition-au-medication.html)|NCTS|
+|[Medication Reason Taken](https://healthterminologies.gov.au/fhir/ValueSet/medication-reason-taken-1)|[AU Base Medication Statement](StructureDefinition-au-medicationstatement.html)|NCTS|
+|[Medication Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-medication-type.html)|[Medication Type](StructureDefinition-medication-type.html)|AU Base|
+|[Medicine Item Change](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-medicine-item-change.html)|[AU Medicine List](StructureDefinition-au-medlist.html)|AU Base|
+|[Medicine Substitution Reason](https://healthterminologies.gov.au/fhir/ValueSet/medicine-substitution-reason-1)|[AU Base Medication Dispense](StructureDefinition-au-medicationdispense.html), [AU Base Medication Request](StructureDefinition-au-medicationrequest.html)|NCTS|
+|[MIMS](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-mims.html)|[AU Base Medication](StructureDefinition-au-medication.html), [AU Base Medication Administration](StructureDefinition-au-medicationadministration.html), [AU Base Medication Dispense](StructureDefinition-au-medicationdispense.html), [AU Base Medication Request](StructureDefinition-au-medicationrequest.html), [AU Base Medication Statement](StructureDefinition-au-medicationstatement.html)|AU Base|
+|[Observation Category Codes - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-observation-category-extended.html)||AU Base|
```

Comment:
Please include Name Context value set (https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-name-context.html), used in Name Context extension (https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-name-context.html) that was only added yesterday.

Reply:
Added, thanks.

Reply:
Confirmed.